libcontainer/cgroups/utils.go

```diff
@@ -50,6 +51,9 @@ func IsCgroup2UnifiedMode() bool {
 
 // https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt
 func FindCgroupMountpoint(cgroupPath, subsystem string) (string, error) {
+	if IsCgroup2UnifiedMode() {
```

Comment:
Can we update godoc comment to make sure `subsystem` is ignored for unified mode? Or can we return `errors.New("subsystem must not be specified for unified mode")`?

Reply:
in another PR.  Trying to keep these changes small so they can be merged.